packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -6,6 +6,8 @@ layout: Guides
 
 Add authorization using Okta to protect your APIs. When you finish, you have a secure REST API application that validates incoming requests.
 
+> **Note:** The [org authorization server](/docs/concepts/auth-servers/#org-authorization-server) isn't supported for Web APIs because your application can't validate the access token issued by an org authorization server. This guide uses the **default** custom authorization server to secure your protected API. You can create your own custom authorization server for this guide. The [Okta Developer Edition](https://developer.okta.com/signup/) makes most key developer features available by default for testing purposes. The [Okta API Access Management product](/docs/concepts/api-access-management/) &mdash; a requirement to use [custom authorization servers](/docs/concepts/auth-servers/#custom-authorization-server) &mdash; is an optional add-on in **production** environments.
```

Comment:
```suggestion
> **Note:** The [org authorization server](/docs/concepts/auth-servers/#org-authorization-server) isn't supported for Web APIs because your application can't validate the access token issued by an org authorization server. This guide uses the **default** custom authorization server to secure your protected API. You can create your own custom authorization server for your app. The [Okta Developer Edition](https://developer.okta.com/signup/) makes most key developer features available by default for testing purposes. The [Okta API Access Management product](/docs/concepts/api-access-management/) &mdash; a requirement to use [custom authorization servers](/docs/concepts/auth-servers/#custom-authorization-server) &mdash; is an optional add-on in **production** environments.
```